app/src/main/java/ch/blinkenlights/android/vanilla/JumpToTimeDialog.java

```diff
@@ -70,9 +82,8 @@ public void onClick(DialogInterface dialog, int which) {
 				int minutes = parseInteger(minutesView.getText().toString());
 				int seconds = parseInteger(secondsView.getText().toString());
 				int position = (hours * 3600 + minutes * 60 + seconds) * 1000;
-				PlaybackService.get(activity).seekToPosition(position);
-				if (activity instanceof SlidingPlaybackActivity) {
-					((SlidingPlaybackActivity) activity).updateElapsedTime();
+				if (activity instanceof OnPositionSubmitListener) {
```

Comment:
I'd remove the if condition here and just crash on line 86 instead of (silently) not performing any action.

LGTM apart from that.